src/main/java/com/google/sps/servlets/utils/CommentAnalysis.java

```diff
@@ -60,7 +61,11 @@ public Statistics computeOverallStats(List<CommentThread> youtubeResponse) {
         youtubeResponse.parallelStream()
```

Comment:
I want to comment on line 58:
This method has an overloaded version: computeOverallStats(List<CommentThread> youtubeResponse, int numTopComments)
Why don't we add the filter there as well?
If they do intend to do almost the same thing, then change this method to simply do:
```
public Statistics computeOverallStats(List<CommentThread> youtubeResponse) {
  return computeOverallStats(youtubeResponse, DEFAULT_TOP_N);
}
```
to avoid duplicated code.